custodial-copy-confirmer/src/main/scala/uk/gov/nationalarchives/confirmer/Main.scala

```diff
@@ -0,0 +1,79 @@
+package uk.gov.nationalarchives.confirmer
+
+import cats.effect.{ExitCode, IO, IOApp}
+import cats.syntax.all.*
+import fs2.Stream
+import pureconfig.ConfigSource
+import uk.gov.nationalarchives.utils.Utils.*
+import pureconfig.*
+import pureconfig.module.catseffect.syntax.*
+import uk.gov.nationalarchives.{DADynamoDBClient, DASQSClient}
+import io.circe.generic.auto.*
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider
+import software.amazon.awssdk.http.nio.netty.{NettyNioAsyncHttpClient, ProxyConfiguration}
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue
+import uk.gov.nationalarchives.DADynamoDBClient.DADynamoDbRequest
+
+import java.net.URI
+import scala.concurrent.duration.*
+import java.util.UUID
+
+object Main extends IOApp {
+
+  case class Config(sqsUrl: String, proxyUrl: URI, ocflRepoDir: String, ocflWorkDir: String) derives ConfigReader
+
+  extension (s: String)
+    private def toAttributeValue: AttributeValue = AttributeValue.builder.s(s).build
+
+  case class Message(dynamoKey: String, dynamoTable: String, ioRef: UUID, batchId: String) {
+    def primaryKey: Map[String, AttributeValue] =
+      Map("assetId" -> ioRef.toString.toAttributeValue, "batchId" -> batchId.toAttributeValue)
+
+    def toUpdate: Map[String, Option[AttributeValue]] =
```

Comment:
Since this doesn't take any parameters, the name looks strange when you call it e.g. `message.toUpdate`; maybe it can be more descriptive like `updateCcStatus`?

Reply:
I've got rid of this because the dynamo attribute is on the config object now.